test-engineering/load-tests/load_test_setup.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+set -eu
+
+#Setup environment
+GCLOUD=$(which gcloud)
+SED=$(which sed)
+KUBECTL=$(which kubectl)
+
+DIRECTORY=$(pwd)
+SDK_DIRECTORY=google-cloud-sdk
+MERINO_DIRECTORY=$DIRECTORY/kubernetes-config
+MASTER_FILE=locust-master-controller.yml
+WORKER_FILE=locust-worker-controller.yml
+SERVICE_FILE=locust-master-service.yml
+
+#Declare variables
+LOCUST_IMAGE_TAG=$(git log -1 --pretty=format:%h)
+echo "Docker image tag for locust is set to: ${LOCUST_IMAGE_TAG}"
+
+##Kubernetes Manifests variables
+KINTO_SERVER_URL=https://firefox.settings.services.mozilla.com
+KINTO_COLLECTION=quicksuggest
+KINTO_BUCKET=main
+CLUSTER=merino-load-test
+TARGET=https://stage.merino.nonprod.cloudops.mozgcp.net
+SCOPE="https://www.googleapis.com/auth/cloud-platform"
```

Comment:
```suggestion
SCOPE=https://www.googleapis.com/auth/cloud-platform
```

Removes quotes to be consistent w/ `KINTO_SERVER_URL` and `TARGET` vars above. Probably only needs quotes in the rare case where there is a `#` in the query string portion of the URL (which YAML would interpret as a comment, because YAML).